content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -1,80 +1,405 @@
+the settings. Navigate to :menuselection:`Settings app`, and under the :guilabel:`Companies`
+section, ensure there is only one company operating the Odoo database. Click :guilabel:`Update Info`
+to open a separate page to update company details.
+
+Should there be multiple companies operating in the database, click :guilabel:`Manage Companies` to
+load a list of companies to select from. Update company information by clicking into the specific
+company.
+
+Database administrators should ensure that the :guilabel:`Street...`, :guilabel:`Street2...`,
+:guilabel:`City`, :guilabel:`State`, :guilabel:`ZIP`, and :guilabel:`Country` are all updated for
+the companies.
+
+This will ensure accurate tax calculations and smooth end-of-year accounting operations.
+
+.. seealso::
+   - :doc:`../../../general/companies`
+   - :doc:`../get_started`
+
+Module installation
+-------------------
+
+Next, ensure that the Odoo *AvaTax* module is installed. To do so, navigate to :menuselection:`Apps
+application`. In the :guilabel:`Search...` bar, type in `avatax`. Press enter and the following
+results populate:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Avatax`
+     - `account_avatax`
+     - Default *AvaTax* module. This module adds the base *AvaTax* features for tax calculation.
+   * - :guilabel:`Avatax for SO`
+     - `account_avatax_sale`
+     - Includes the information needed for tax calculation on sales orders in Odoo.
+   * - :guilabel:`Avatax for Subscriptions`
+     - `account_avatax_sale_subscription`
+     - This module includes the features required for tax calculation on subscriptions in Odoo.
+   * - :guilabel:`Account Avatax - Ecommerce`
+     - `website_sale_account_avatax`
+     - Includes tax calculation features for the checkout process on Odoo eCommerce.
+   * - :guilabel:`Account AvaTax - Ecommerce - Delivery`
+     - `website_sale_delivery_avatax`
+     - Includes tax calculation features for the delivery process on Odoo eCommerce.
+
+Click on the :guilabel:`Install` button on the module labeled :guilabel:`Avatax`: `account_avatax`.
+This will install the following modules:
+
+- :guilabel:`Avatax`: `account_avatax`
+- :guilabel:`Avatax for SO`: `account_avatax_sale`
+- :guilabel:`Account Avatax - Ecommerce`: `website_sale_account_avatax`
+
+Should *AvaTax* be needed for Odoo *Subscriptions* or for delivery tax in Odoo *Ecommerce*, then
+install those modules individually by clicking on :guilabel:`Install`.
+
+.. _avatax/credentials:
+
+Odoo AvaTax settings
+--------------------
+
+To integrate the *AvaTax* :abbr:`API (application programming interface)` with Odoo, go to
+:menuselection:`Accounting --> Configuration --> Settings` section. The :guilabel:`AvaTax` fields in
+the :guilabel:`Taxes` section is where the *AvaTax* configurations are made and the credentials are
```

Comment:
```suggestion
the :guilabel:`Taxes` section is where the *AvaTax* configurations are made, and the credentials are
```

Reply:
It might be smart to mention that the user must tick the checkbox for AvaTax in order for the other fields to appear.

Reply:
This checkbox is actually for the feature to the right of the AvaTax settings.
![2024-05-09_13-12](https://github.com/odoo/documentation/assets/91896312/4bd57e4d-2c06-4242-ace9-396bf0ac84c3)

Reply:
Update- this is for Odoo 17 that you need to tick the checkbox. I will add that in